data/root/gdb.file_list

```diff
@@ -3,6 +3,8 @@ TEMPLATE:
 
 gdb:
 ?valgrind: nodeps
+# helps valgrind download debuginfos
+?debuginfod-client:
```

Comment:
This is the one-liner feature,
the rest of the PR are tests added in an attempt to debug a transient build failure :rofl: